.github/workflows/ci.yml

```diff
@@ -66,6 +66,13 @@ jobs:
           restore-keys: |
             app_home_lint-${{ matrix.php-version }}
             app_home_lint-
+      - name: "Restore PHPStan cache"
+        uses: actions/cache/restore@v3
+        with:
+          path: tmp
+          key: "phpstan-result-cache-${{ github.run_id }}"
+          restore-keys: |
+            phpstan-result-cache-
```

Comment:
Instead of adding two new steps, you can just add the `/tmp/phpstan` path in the `Restore lint cache` step `path` value.